test/verify/test_step_trans.cpp

```diff
@@ -0,0 +1,22 @@
+
+#include "verify_program.hpp"
+#include <migraphx/program.hpp>
+#include <migraphx/generate.hpp>
+#include <migraphx/make_op.hpp>
+
+struct test_step_trans : verify_program<test_step_trans>
```

Comment:
This should probably be `test_step_transpose`

Reply:
fixed